apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -220,15 +220,22 @@ app = do
         content = if isNothing state.activeUser && needsLogin route
                   then Views.loginView { state, setState } (setUser (Nothing :: Maybe Days)) logger
                   else userContent
-    pure $ render self router (foldMap Elements.loadingIndicator state.loading <> content) logout isPersonating
+        navbarView :: JSX
```

Comment:
I think the context is simple enough for this to be okay without stating the type.